packages/hooks/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@aries-framework/hooks",
+  "version": "0.0.1",
+  "description": "This package exposes useful hooks that allow you to easily interact with AFJ.",
+  "main": "index.tsx",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "Apache-2.0",
+  "peerDependencies": {
+    "@aries-framework/core": "0.1.0-alpha.273",
+    "@aries-framework/react-native": "0.1.0-alpha.273",
```

Comment:
Could you update it to the same as other packages in this repo (276)?

Also, peerDependencies should be listed in devDependencies (see redux-store as example). Also, I don't think it is strictly required to have the react native package in here, it's only using the core package